docs/no_peeking/mappings/patient.py

```diff
@@ -0,0 +1,54 @@
+def individual_texonomy(x):
+    """
+    https://aehrc.github.io/fhir-phenopackets-ig/ValueSet-taxonomy.html
+    """
+    if x == constants.SPECIES.DOG:
+        return {
+            'code': 'NCBITaxon_9615',
+            'display': 'Canis lupus familiaris'
+        }
+    elif x == constants.SPECIES.HUMAN:
+        return {
+            'code': 'NCBITaxon_9606',
+            'display': 'Homo sapiens'
+        }
+    else: 
+        raise Exception('Unknown Patient individual-taxonomy')
+
+def gender(x):
+    """
+    http://hl7.org/fhir/R4/valueset-administrative-gender.html
+    """
+    if x == constants.GENDER.FEMALE:
+        return 'female'
+    elif x == constants.GENDER.MALE:
+        return 'male'
+    elif x == constants.COMMON.OTHER:
+        return 'other'
+    elif x == constants.COMMON.UNKNOWN:
+        return 'unknown'
+    else:
+        raise Exception('Unknown Patient gender')
+
+
+patient = {
+    'resourceType': 'Patient'
+    'id': PARTICIPANT.TARGET_SERVICE_ID,
```

Comment:
target service ID isn't known before it goes into the server. what do we do about this?

Reply:
Yeah, the use of `TARGET_SERVICE_ID` is like place-holding at the moment. As far as I understand, the `'id'` field is the one used when we hit each of the REST endpoints like `BASE_URL/Patient/id`. So, it seems like we need to define some consistent, global scheme of `'id'`. Any thoughts on this, @znatty22 and @ShahimEssaid?